mealie/routes/users/api_tokens.py

```diff
@@ -15,17 +15,22 @@ class UserApiTokensController(BaseUserController):
     @router.post("/api-tokens", status_code=status.HTTP_201_CREATED, response_model=LongLiveTokenOut)
     def create_api_token(
         self,
-        token_name: LongLiveTokenIn,
+        token_params: LongLiveTokenIn,
     ):
         """Create api_token in the Database"""
 
-        token_data = {"long_token": True, "id": str(self.user.id)}
+        token_data = {
+            "long_token": True,
+            "id": str(self.user.id),
+            "name": token_params.name,
+            "integration_id": token_params.integration_id,
+        }
```

Comment:
I know adding this to the JWT was my idea, but I'm wondering if we _need_ that validation that using a JWT provides? Is there any security reason we couldn't just expose a `Mealie-Integration-Id` header and allow any user to provide it and just use the header instead of trying to grab the integration_id out of the JWT? 

I think that would simplify things a bit, but I'm not sure if there's a case I'm not considering.

Reply:
My thought with the JWT was that you can add it to your token and just forget about it, that way you don't have to add it to each call. Otherwise, I don't see an issue with using a header instead. Is there an easy way to add it to every route as a header param, like at the router level?

Another option would be to ditch the integration id entirely and just rely on the token name, though I feel dirty using the name for anything other than a human-friendly label.

Reply:
Since you've already done the work to put it in the JWT, I'm okay with going down that road. The overall difference between the JWT and header is pretty minimal so realistically it probably doesn't matter. It is probably more ergonomic for clients to not have to think about that.